Functions/InputOutput/xASL_io_dcm2nii.m

```diff
@@ -10,7 +10,7 @@
 %      destdir         target destination directory (REQUIRED)
 %	   series_name     target name of the NIfTI file (REQUIRED)
 %      imPar           imPar from x.modules.import.imPar (OPTIONAL, DEFAULT = [])
-%                      'imPar.dcm2nii_version' (DEFAULT 20220720)
+%                      'imPar.dcm2nii_version' (DEFAULT 20240202)

```

Comment:
Did you test with all flavors that the new version works? We also have/had some exceptions where we defaulted to an older dcm2nii version, e.g., for PAR/REC, which was supported in an older version but not in the (back then) newer version.

Reply:
Yes, all flavors work. PAR/REC of Look-Locker needs to use the old version. All others work nicely.